virtcontainers/qemu.go

```diff
@@ -646,17 +646,19 @@ func (q *qemu) stopSandbox() error {
 
 	err := q.qmpSetup()
 	if err != nil {
-		return err
+		q.Logger().WithError(err).Error("Failed to setup qmp")
 	}
 
 	err = q.qmpMonitorCh.qmp.ExecuteQuit(q.qmpMonitorCh.ctx)
 	if err != nil {
```

Comment:
At this point it still continue to delete  VM paths that the VM process is using them, if there quit qmp failed and we are still forcing to continue, should not be ensure that we force to kill the VM?  @jodh-intel @grahamwhaley any suggestion about it ?